Signal/src/call/CallService.swift

```diff
@@ -281,6 +281,7 @@ protocol CallServiceObserver: class {
             peerConnectionClient.createSignalingDataChannel()
 
             assert(self.peerConnectionClient == nil, "Unexpected PeerConnectionClient instance")
+            Logger.debug("\(self.TAG) setting peerConnectionClient in \(#function)")
```

Comment:
I've also added some logging around the setting of the pcc property. It should help us diagnose the "unexpected pcc" issue the next time we repro it.